src/justenoughicu.c

```diff
@@ -33,13 +32,10 @@ int icu_case(lua_State *L) {
   int32_t l2 = 0;
 
   if (strcmp(recase, "title") == 0) {
-    conversion_function_ti conversion;
-    UBreakIterator *iter;
-    conversion = u_strToTitle;
-    l2 = conversion(NULL, 0, input_as_uchar, l, iter, locale, &err);
+    l2 = u_strToTitle(NULL, 0, input_as_uchar, l, NULL, locale, &err);
```

Comment:
Thanks @srl295. I'm afraid I was out of my depth just hacking this feature in, the optimization might have to come from somebody with better bigger C chops. On the other hand your tip on the iterator did stop it from blowing up in my face, so that's nice.